code/game/objects/items/theft_items.dm

```diff
@@ -171,6 +172,10 @@
 	return
 
 /obj/item/nuke_core/supermatter_sliver/can_be_pulled(user) // no drag memes
+	if(ishuman(user))
+		var/mob/living/carbon/human/H = user
+		if(HAS_TRAIT(H, TRAIT_SUPERMATTER_IMMUNE))
+			return TRUE
 	return FALSE
```

Comment:
why do you need to check for human here?

Reply:
i'm honestly not sure why I added that check, removing in the next commit